api/base/serializers.py

```diff
@@ -7,12 +7,17 @@
 
 
 def _rapply(d, func, *args, **kwargs):
-    """Apply a function to all values in a dictionary, recursively."""
+    """Apply a function to all values in a dictionary, recursively. Handles lists and dicts currently,
+    as those are the only complex structures currently supported by DRF Serializer Fields."""
     if isinstance(d, collections.Mapping):
         return {
             key: _rapply(value, func, *args, **kwargs)
             for key, value in d.iteritems()
         }
+    if isinstance(d, list):
```

Comment:
All that said, since Sam is replacing this in his PR, we can probably just defer these changes.